pkg/agent/metrics/prometheus.go

```diff
@@ -146,6 +146,16 @@ var (
 		},
 	)
 
+	TotalReconnectionsToFlowCollector = metrics.NewGauge(
```

Comment:
I'm following the naming ways of other variables. Do you think `ReconnectionsToFlowCollector` is better?

Reply:
For some of the existing metrics, the `Total` prefix makes sense because we have 2 different metrics (one for the total count, and one which gives the count based on a specific category). It's not the case for this new metric.